.teamcity/src/main/kotlin/configurations/Gradleception.kt

```diff
@@ -20,15 +21,31 @@ class Gradleception(model: CIBuildModel, stage: Stage) : BaseGradleBuildType(sta
         javaCrash = false
     }
 
+    params {
+        // Override the default commit id so the build steps produce reproducible distribution
+        param("env.BUILD_COMMIT_ID", "HEAD")
+    }
+
+    // So that the build steps produces reproducible distribution
+    val magicTimestamp = "20210102030405+0000"
```

Comment:
If we use this magic timestamp, then we would need to remove the caches for that version from the user-home before the second build. I'd actually prefer to try to determine the md5 hash for the first build by running a separate build before, which has a task to only determine the hash of the distribution and write it to a file. Then we could use that hash as an input for building the first distribution.